src/hotspot/share/runtime/os.hpp

```diff
@@ -404,6 +405,9 @@ class os: AllStatic {
   // Return the default page size.
   static size_t vm_page_size() { return OSInfo::vm_page_size(); }
 
+  static size_t align_up_vm_page_size(size_t size)   { return align_up  (size, os::vm_page_size()); }
```

Comment:
Style Nit: please don't add extra whitespace before the opening function parenthesis.

Reply:
No not a strong request. I see there is precedent for this already in the file - though in some cases it isn't even needed for alignment purposes. I find extra whitespace between a function name and the opening parenthesis to be visually jarring and it makes me have to re-read to ensure I'm not mis-reading something. YMMV.